### ISSUE TITLE
Cannot find module 'lodash/array' error

### ISSUE THREAD

**Issue by luzfcb** [xavdid/heroku-config#10]
hello, I tried to install heroku-config today, however, occurred this error. I suppose that is missing a dependency:

```
fabio@luzfcb:~$ heroku plugins:install heroku-config
Installing plugin heroku-config...module.js:442
    throw err;
    ^

Error: Cannot find module 'lodash/array'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/luzfcb/.local/share/heroku/plugins/node_modules/heroku-config/commands/push.js:7:15)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
 !
 ▸    Error installing plugin heroku-config
 !    error installing plugin heroku-config
```

**Comment by xavdid** [comment]
That's no good! that was caused by some dependency changes that worked in test, but apparently not in production. I've pushed a fix (`v1.3.1`) and you should be able to install properly now. 

Thanks for flagging!

**Comment by luzfcb** [comment]
@xavdid wow... very quick fix. thank a lot.
